config/locales/certificates.en.yml

```diff
@@ -39,13 +40,19 @@ en:
         broker_dealer: "broker and dealer"
         carrier_broker_dealer: "carrier, broker and dealer"
       confirm_email:
-        legend: "Confirm your email address"
+        title: "In order to access your certificate, enter the email address that was used during registration."
+        heading: "In order to access your certificate, enter the email address that was used during registration."
+        email_label: "Email address"
+        hint: "This is the email address that was entered during registration"
+        submit: "Confirm"
       process_email:
-        error: "Invalid email address"
+        error: "You need to enter the email address that the certificate link was sent to."
       renew_token:
+        title: "The certificate link has expired, enter an email address that was used during registration."
```

Comment:
Not a big issue, but generally we don't put dots at the end of the sentence (only when you have several sentences and you need to separate them)

Reply:
Do you mean for error messages? There are several parts wireframes in this ticket which include them such as https://ccnl83.axshare.com/the_certificate_link_has_expired__enter_the_email_.html

Reply:
Bit inconsistent, but yeah, I see now.